src/renderers/shared/stack/reconciler/__tests__/ReactErrorBoundaries-test.js

```diff
@@ -84,12 +103,15 @@ describe('ReactErrorBoundaries', function() {
       }
     }
 
-    var EventPluginHub = require('EventPluginHub');
     var container = document.createElement('div');
-    EventPluginHub.putListener = jest.fn();
     container.innerHTML = ReactDOMServer.renderToString(<Boundary />);
     expect(container.firstChild.innerHTML).toBe('Happy Birthday!');
-    expect(EventPluginHub.putListener).not.toBeCalled();
+    // note: renderToString does not call componentDidMount
```

Comment:
Should we have tests for `renderToString` and `ReactDOM.render`? The [title of this test block](https://github.com/facebook/react/pull/7569/files#diff-2a70af57f7fd1fa492c1f734ee9b452eR62) should be changed too if we're not testing SSR.